### PR TITLE
Added installation instructions to readme

### PR DESCRIPTION
**Pull request by JeremybellEU**
Added the installation instructions to the readme. This should explain new developers how to contribute to the project.
Please verify for both IDE's that this setup is correct and working.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,23 @@
 [![Stories in Ready](https://badge.waffle.io/MoodCat/MoodCat.me.svg?label=ready&title=Ready)](http://waffle.io/MoodCat/MoodCat.me)
 
 [![Throughput Graph](https://graphs.waffle.io/MoodCat/MoodCat.me/throughput.svg)](https://waffle.io/MoodCat/MoodCat.me/metrics)
+
+# Installation instructions
+
+To succesfully run/test the application, a couple of steps should be followed to install various dependencies and configurations:
+
+1. After cloning, open the IDE of your preference (Eclipse/IntelliJ), import the project as maven project and make sure `target/metamodel` is a resource-folder of your project.
```
Comment (JWGmeligMeyling) [inline README.md]: I should be a `source folder` in Eclipse, not `resource folder`.

File: README.md [hunk 2]
```diff
@@ -4,3 +4,23 @@
...
+3. Install Lombok
+  * (Eclipse only) Close your IDE and run the following jar. `.m2/repository/org/projectlombok/lombok/LATEST_LOMBOK_VERSION/lombok-LATEST_LOMBOK_VERSION.jar`
+The installation will provide the steps to attach Lombok to your IDE.
+  * (IntelliJ only) Go to `preferences->plugins->browse repositories->lombok` and install the plugin.
+4. (Production only) Whenever you want to run the App in production, the database connection requires a password. Therefore it is not possible to launch if you don't add this password to the environment:
+    * (Eclipse only) Go to `run->run configurations->Java Application->App`.
+Click the `Environment` tab.
+    * (IntelliJ only) Go to `me.moodcat.core.App` and click on `edit configurations`.
+  * Click on `new`.
+As name set `database-password` and as value the database password.
```
Comment (JWGmeligMeyling) [inline README.md]: You can also run the main from the test package and use the embedded H2 database instead.

**Comment by Azkle** [comment]
Looks good to me.

File: README.md (hunk 2 above)
Comment (JeremybellEU) [inline README.md]: Does this use the correct persistence.properties? As it seems to me (after running) that it just runs the core with the inserted password.

**Comment by JeremybellEU** [comment]
I rebased this branch onto master to get some more updates. Added some more clarification according to comments and also added a new exception to notify developers about misconfiguration.

**Comment by GijsWeterings** [comment]
LGTM

**Pull request merged**